src/plopp/widgets/drawing.py

```diff
@@ -76,10 +76,10 @@ def make_node(self, artist):
         output_node.name = f'Output node {len(self._output_nodes)}'
         self._output_nodes[nodeid] = output_node
         if self._destination_is_fig:
-            output_node.add_view(self._destination)
+            output_node.add_view(self._destination._fig)
             self._destination.update(new_values=output_node(), key=output_node.id)
-            self._destination.artists[output_node.id].color = getattr(
-                artist, 'color', artist.edgecolor)
+            self._destination.artists[output_node.id].color = artist.color if hasattr(
+                artist, 'color') else artist.edgecolor
```

Comment:
Is this something backend specific?

Reply:
No, it depends which shapes you are drawing. A line will have a `color`, while a rectangle will have a `facecolor` and an `edgecolor`.
All this can only be achieved with the matplotlib backend, because `mpltoolbox` is Matplotlib specific. You cannot use `plotly` here.